src/mmrelay/cli.py

```
@@ -1545,7 +1555,7 @@ def handle_auth_login(args: argparse.Namespace) -> int:
             from mmrelay.log_utils import get_logger
 
             logger = get_logger("CLI")
```

Ruff (0.14.14)

[error] Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)